liquibase-core/src/main/java/liquibase/command/core/HistoryFormat.java

```diff
@@ -0,0 +1,5 @@
+package liquibase.command.core;
+
+public enum HistoryFormat {
+    TEXT, TABULAR
```

Comment:
Teensie-tiny little nit ... can we put the ENUM values on separate rows, please?

Reply:
Done